.github/workflows/deploy.yml

```diff
@@ -0,0 +1,29 @@
+name: GitHub Pages
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@master
+  
+    - name: Install Dependencies
+      run: npm ci
+
+    - name: Build
+      run: npm run build
+      if: github.event.deleted == false
+
+    - name: Deploy
+      uses: peaceiris/actions-gh-pages@v1.1.0
+      if: success()
```

Comment:
Isn't that the default anyway?

Reply:
I think so, but the example code had that if statement and didn't set `continue-on-error` to true, so... 🤷‍♂

Reply:
Yep, it's necessary, because the previous step is executed conditionally, and if it doesn't execute, its status is "neutral".

Full explanation: https://github.com/peaceiris/actions-gh-pages/commit/6492b522cec8813590f3f31094b75c43725adccf#commitcomment-34950365